app/commands/user/reputation_token/award_for_pull_request.rb

```diff
@@ -22,6 +23,26 @@ def award_reputation_to_author
         User::ReputationToken::CodeContribution::Create.(user, external_link, repo, number, reason)
       end
 
+      def award_reputation_to_reviewers
+        return unless closed?
```

Comment:
I think we only award reputation once the PR is merged. So should we guard on line 15 too?

Reply:
Actually, should we just guard the whole thin on L11?

Reply:
Yes. I hadn't yet gotten around the merged bit. I've updated